vignettes/datatable-reference-semantics.Rmd

```diff
@@ -68,7 +68,7 @@ DF$c <- 18:13               # (1) -- replace entire column
 DF$c[DF$ID == "b"] <- 15:13 # (2) -- subassign in column 'c'
 ```
 
-both (1) and (2) resulted in deep copy of the entire data.frame in versions of `R` versions `< 3.1`. [It copied more than once](https://stackoverflow.com/q/23898969/559784). To improve performance by avoiding these redundant copies, *data.table* utilised the [available but unused `:=` operator in R](https://stackoverflow.com/q/7033106/559784).
+both (1) and (2) resulted in deep copy of the entire data.frame in versions of `R` `< 3.1`. [It copied more than once](https://stackoverflow.com/q/23898969/559784). To improve performance by avoiding these redundant copies, *data.table* utilised the [available but unused `:=` operator in R](https://stackoverflow.com/q/7033106/559784).
```

Comment:
`R` `< 3.1` probably should be combined to `R < 3.1` now that there is nothing between them

I'd personally not consider the version number to be something within backticks, but to maintain consistency with cases like `R 3.1+` below:
https://github.com/Rdatatable/data.table/blob/a673460dd83fa499edee71d7eee7fa1f1b5f3fa6/vignettes/datatable-reference-semantics.Rmd#L73